test/inputs/delete-by-name/budgetadjustment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   displayName: NewRelic budget adjustment
 spec:
   description: Example budget adjustment
-  firstEventStart: 2024-03-05T05:00:00Z
+  firstEventStart: <FUTURE_TIME>
```

Comment:
Shouldn't this be `<NEXT_DAY_TIME>`?

Reply:
fixed in https://github.com/nobl9/sloctl/pull/160/commits/841badf2fdd1d53ded25d364363aba38b9f5d633